.coveragerc

```
@@ -10,6 +10,6 @@ omit =
      */imaginary/pyparsing.py
 
 [paths]
-source =
+paths =
```

If this was desired to be unchanged.

```suggestion
source =
```

---

Heh.  Woops.  Oh well, not a big deal.